docs/create_own_rule.md

```diff
@@ -115,15 +115,23 @@ composer dump-autoload
 
 ## 2. Register It
 
-```yaml
-# rector.yaml
-services:
-    Utils\Rector\MyFirstRector: ~
+```php
+<?php
+// rector.php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(Utils\Rector\MyFirstRector::class);
```

Comment:
@TomasVotruba Not sure if the services configuration is done correctly here.

Reply:
Bette use full import and here only short one, to be consistent
```suggestion
    $services->set(MyFirstRector::class);
```

Reply:
What exactly do you mean?

Reply:
Oh, I might see what you mean. Here only stirng short path should be used. The class doesn't exist yet and the path will be handled by Rector. Programmers don't have to think about it 

```suggestion
    $services->set('MyFirstRector');
```

Reply:
It the same as it was in yaml example?
```yaml
services:
    Utils\Rector\MyFirstRector: ~
```

Reply:
I confused it with config. Yours is right.
Just import the FQN